.github/workflows/buildfrontend.yml

```diff
@@ -82,6 +82,22 @@ jobs:
           app_tag=$(echo $app_tag | sed 's/^"//;s/"$//')
           sed -i "s/app.js/$app_tag/" 'resources/public/index.html'
 
+      - name: Conditionally update base URLs in index (master)
+        if: endswith(github.ref, '/master')
+        run: |
+          sed -i 's/(?<=api_url_base = ").+(?=";)/https:\/\/data.imas.utas.edu.au\/seamap\/api\//' 'resources/public/index.html'
+          sed -i 's/(?<=media_url_base = ").+(?=";)/https:\/\/data.imas.utas.edu.au\/seamap\/media\//' 'resources/public/index.html'
+          sed -i 's/(?<=wordpress_url_base = ").+(?=";)/https:\/\/seamapaustralia.org\//' 'resources/public/index.html'
+          sed -i 's/(?<=img_url_base = ").+(?=";)/\/app\/img\//' 'resources/public/index.html'
+
+      - name: Conditionally update base URLs in index (develop)
+        if: endswith(github.ref, '/develop')
+        run: |
+          sed -i 's/(?<=api_url_base = ").+(?=";)/https:\/\/data-dev.imas.utas.edu.au\/seamap\/api\//' 'resources/public/index.html'
+          sed -i 's/(?<=media_url_base = ").+(?=";)/https:\/\/data-dev.imas.utas.edu.au\/seamap\/media\//' 'resources/public/index.html'
+          sed -i 's/(?<=wordpress_url_base = ").+(?=";)/https:\/\/seamapaustralia-dev.imas.utas.edu.au\//' 'resources/public/index.html'
+          sed -i 's/(?<=img_url_base = ").+(?=";)/\/app\/img\//' 'resources/public/index.html'
+
```

Comment:
Aah -- thanks for that, but the goal is we don't need them here either (this is effectively still the issue that we have a deployment per branch, that we're trying to move away from).  We'll have basically this code still, but run at deploy-time via ansible, instead of github-actions.

Reply:
Did you want me to leave it in for now, to be removed when the ansible deployment code has been updated, or has that all ready been taken care of?

Reply:
Yeah, I guess leave it in thanks, then it's easily copyable for the playbook